.github/workflows/ci.yaml

```
@@ -110,6 +110,56 @@ jobs:
           name: Artefacts
           path: artefacts.tar
 
+  build_storybook:
+    runs-on: ubuntu-latest
+    steps:
+      # Using v1 (rather than v2) through-out this workflow due to issue:
+      # https://github.com/actions/checkout/issues/237
+      - uses: actions/checkout@v1
+
+      - name: NPM Cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-
```

It would be prudent to have it's own cache for this me thinks.

be good to replace `-npm-` on lines 124 and 126:

```suggestion
          key: ${{ runner.os }}-npm.storybook-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-npm.storybook-
```